WordPress/Classes/ViewRelated/Blog/My Site/MySiteViewController.swift

```diff
@@ -0,0 +1,255 @@
+import CoreData
+import Foundation
+
+class MySiteViewController: UIViewController, NoResultsViewHost {
+
+    // MARK: - Initializers
+
+    init(meScenePresenter: ScenePresenter) {
+        self.meScenePresenter = meScenePresenter
+
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    required init?(coder: NSCoder) {
+        meScenePresenter = MeScenePresenter()
```

Comment:
I understand this if we want to still allow to instantiate this via storyboard, but if for any reason we substitute the `ScenePresenter` concrete type, this will need to be handled manually

Reply:
I could make this fatally fail if you prefer.  I'm ok either way since this initialiser isn't used.

Let me know which one feels best.

Reply:
we could make it fail